library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java

```diff
@@ -213,7 +226,14 @@ public void onClick(View view) {
                 mRadius = mCalculator.getViewRadius();
             }
 
+
             imageView.setParameters(mBackgroundColor, mCalculator);
+            if (mFocusPositionX > 0 && mFocusPositionY > 0 && mFocusRectangleWidth > 0 && mFocusRectangleHeight > 0) {
```

Comment:
You force position values to be bigger than 0. I think -1 would be better. Because there may be a need to focus to left/top edge of the device.